sub-packages/bionemo-geneformer/src/bionemo/geneformer/data/singlecell/datamodule.py

```diff
@@ -191,7 +193,6 @@ def _create_dataloader(self, dataset, **kwargs) -> DataLoader:
                 min_length=None,
                 max_length=self.max_len,
             ),
-            **kwargs,
```

Comment:
You can still pass `**kwargs` through incase there's anything else a user wants to pass in? Not saying we should remove this.